lib/ansible/modules/identity/cyberark/cyberark_user.py

```diff
@@ -0,0 +1,515 @@
+    initial_password: "password"
+    user_type_name: "EPVUser"
+    change_password_on_the_next_logon: false
+    state: present
+    cyberark_session: "{{ cyberark_session }}"
+
+- name: Add User to Group "GroupOfUsers"
+  cyberark_user:
+    username: "username"
+    group_name: "GroupOfUsers"
+    state: addtogroup
+    cyberark_session: "{{ cyberark_session }}"
+
+- name: Reset user credential
+  cyberark_user:
+    username: "Username"
+    new_password: "password"
+    disabled: false
+    state: update
+    cyberark_session: "{{ cyberark_session }}"
+
+- name: Logoff from CyberArk Vault
+  cyberark_authentication:
+    state: absent
+    cyberark_session: "{{ cyberark_session }}"
+'''
+
+RETURN = '''
+cyberark_user:
+    description: Dictionary containing result properties.
+    returned: success
+    type: dict
+    sample:
+        AgentUser:
+            description: Whether or not this user is a gateway user.
+            type: bool
+            returned: always
+        Disabled:
+            description: Whether or not the activated user is disabled.
+            type: bool
+            returned: always
+        Email:
+            description: User email address.
+            type: string
+            returned: always
+        Expired:
+            description: Whether or not the user password has expired.
+            type: bool
+            returned: always
+        ExpiryDate:
+            description: The date when the user account will expire and become disabled.
+            type: string
+            returned: always
+        FirstName:
+            description: The user first name.
+            type: string
+            returned: always
+        LastName:
+            description: The user last name.
+            type: string
+            returned: always
+        Location:
+            description: The Location of the user in the Vault hierarchy.
+            type: string
+            returned: always
+        Source:
+            description: Whether user was created internally, or is an external user who was created from an LDAP directory.
+            type: string
+            returned: always
+        Suspended:
+            description: Whether or not the user is suspended.
+            type: bool
+            returned: always
+        username:
+            description: The name of the user.
+            type: string
+            returned: always
+        UserTypeName:
+            description: The type of the user.
+            type: string
+            returned: always
+status_code:
+    description: Result HTTP Status code
+    returned: success
+    type: int
+    sample: 200
+'''
+
+from ansible.module_utils._text import to_text
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.urls import open_url
+from ansible.module_utils.six.moves.urllib.error import HTTPError
+import json
+try:
+    import httplib
+except ImportError:
+    # Python 3
+    import http.client as httplib
```

Comment:
note: standard python style is to make three groups of imports.  libraries from the standard library first, then imports from third party libraries, and finally imports from the project you're a part of.  imports should also be roughly alphabetical.  So the order of imports should look more like this:
``` python
import json
try:
    import httplib
except ImportError:
    # Python 3
    import http.client as httplib

from ansible.module_utils.basic import AnsibleModule
from ansible.module_utils._text import to_text
from ansible.module_utils.six.moves.urllib.error import HTTPError
from ansible.module_utils.urls import open_url
```

(httpclient, because it has the try except block is not included in the alphabetical sorting... see my comment specifically on http.client though.)